test/libethereum/BlockChainTests.cpp

```diff
@@ -60,9 +60,30 @@ struct ChainBranch
 			importedBlocks.pop_back();
 	}
 	TestBlockChain blockchain;
-	vector<TestBlock> importedBlocks;
+	vector<TestBlock> importedBlocks;	
+
+	static void forceBlockchain(string const& chainname)
+	{
+		tempBlockchainNetwork = dev::test::TestBlockChain::sealEngineNetwork;
+		if (chainname == "Frontier")
+			dev::test::TestBlockChain::sealEngineNetwork = eth::Network::FrontierTest;
```

Comment:
`dev::test::TestBlockChain::sealEngineNetwork` should be `dev::test::TestBlockChain::g_sealEngineNetwork`.